sanic/app.py

```diff
@@ -570,9 +570,10 @@ def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
                 serve(**server_settings)
             else:
                 serve_multiple(server_settings, workers)
-        except:
+        except Exception:
```

Comment:
I would think if you want to catch everything, using

```python
try:
    ...
except BaseException:
    ...
```

is better than bare `except:` as it is much more clear.